src/pages/settings/Profile/ProfilePage.js

```diff
@@ -19,189 +17,102 @@ import * as PersonalDetails from '../../../libs/actions/PersonalDetails';
-                        pendingAction={lodashGet(this.props.currentUserPersonalDetails, 'pendingFields.avatar', null)}
-                        errors={lodashGet(this.props.currentUserPersonalDetails, 'errorFields.avatar', null)}
-                        errorRowStyles={[styles.mt6]}
-                        onClose={PersonalDetails.clearAvatarErrors}
-                    >
-                        <AvatarWithImagePicker
-                            isUsingDefaultAvatar={ReportUtils.isDefaultAvatar(lodashGet(currentUserDetails, 'avatar', ''))}
-                            source={ReportUtils.getAvatar(lodashGet(currentUserDetails, 'avatar', ''), lodashGet(currentUserDetails, 'login', ''))}
-                            onImageSelected={PersonalDetails.updateAvatar}
-                            onImageRemoved={PersonalDetails.deleteAvatar}
-                            anchorPosition={styles.createMenuPositionProfile}
-                            size={CONST.AVATAR_SIZE.LARGE}
-                        />
-                    </OfflineWithFeedback>
-                    <View style={[styles.mt4]}>
-                        {_.map(profileSettingsOptions, (detail, index) => (
-                            <MenuItemWithTopDescription
-                                key={`${detail.title}_${index}`}
-                                shouldShowRightIcon
-                                title={detail.title}
-                                description={detail.description}
-                                onPress={() => Navigation.navigate(detail.pageRoute)}
-                            />
-                        ))}
-                    </View>
-                    <LoginField
-                        label={this.props.translate('profilePage.emailAddress')}
-                        type="email"
-                        login={this.state.logins.email}
-                        defaultValue={this.state.logins.email}
-                    />
-                    <LoginField
-                        label={this.props.translate('common.phoneNumber')}
-                        type="phone"
-                        login={this.state.logins.phone}
-                        defaultValue={this.state.logins.phone}
-                    />
-                    <MenuItem
-                        title={this.props.translate('privatePersonalDetails.personalDetails')}
-                        icon={Expensicons.User}
-                        onPress={() => Navigation.navigate(ROUTES.SETTINGS_PERSONAL_DETAILS)}
-                        shouldShowRightIcon
+        return lodashGet(props.translate('pronouns'), pronounsKey, props.translate('profilePage.selectYourPronouns'));
+    };
+    const currentUserDetails = props.currentUserPersonalDetails || {};
+    const profileSettingsOptions = [
+        {
+            description: props.translate('displayNamePage.headerTitle'),
+            title: lodashGet(currentUserDetails, 'displayName', ''),
```

Comment:
Hey 👋, coming from https://github.com/Expensify/App/issues/72240
If user uses a phone number as a login, `displayName` is a phone number, which means it will end with `@expensify.sms`
Same as we do it for the phone number below, this should have been wrapped with `Str.removeSMSDomain` (`formatPhoneNumber` today)